psydac/api/tests/test_api_glt_2d_scalar.py

```diff
@@ -6,7 +6,7 @@
 from sympde.core import Constant
 from sympde.calculus import grad, dot, inner, cross, rot, curl, div
 from sympde.calculus import laplace, hessian
-from sympde.topology import (dx, dy, dz)
+from sympde.topology import dx, dy, dz, dx1, dx2, dx3
```

Comment:
Please remove the import of `(dx, dy, dz)`, as they are never used in this file.

Reply:
done